src/function.rs

```diff
@@ -411,3 +415,14 @@ fn without_extension(_context: &FunctionContext, path: &str) -> Result<String, S
 
   Ok(parent.join(file_stem).to_string())
 }
+
+/// Check whether a string processes properly as semver (e.x. "0.1.0")
+/// and matches a given semver requirement (e.x. ">=0.1.0")
+fn semver_matches(_context: &FunctionContext, v: &str, req: &str) -> Result<String, String> {
```

Comment:
Maybe a little more explicit:

```suggestion
fn semver_matches(_context: &FunctionContext, version: &str, requirement: &str) -> Result<String, String> {
```